### DISCUSSION TITLE
How to evaluate with a different settings of "small" objects?

### DISCUSSION THREAD

**Discussion by le02146** [rafaelpadilla/review_object_detection_metrics#128] [Q&A]
The coco evaluation metrics use AP in three different size scales: small ( `area < 32^2 pixels `), medium and large.  
https://github.com/rafaelpadilla/review_object_detection_metrics#aps-apm-and-apl- 
**How to use dfferent setting?**

In my case, I want to define a object  as small when the **width ratio** is small ?
For example, `width_of_bbox / width_of_image < 10%`  . 
Any suggestioins?
Thank you .

**Comment by rafaelpadilla** [comment]
Hi @le02146 ,

By default, the scale metrics use absolute ares (in pixels). For the relative/ratio, you'll need to implement the logic in the function [_evaluate_image(...)](https://github.com/rafaelpadilla/review_object_detection_metrics/blob/main/src/evaluators/coco_evaluator.py#L300).
